src/toolkitAPI.c

```diff
@@ -1090,6 +1090,59 @@ int DLLEXPORT swmm_getSystemRunoffStats(TRunoffTotals *runoffTot)
     return(errorcode);
 }
 
+int DLLEXPORT swmm_getGageParam(int index, int Param, double *value)
+//
+// Input:   index = Index of desired ID
+//          param = Parameter desired (Perhaps define enum)
+// Output:  value = value to be output
+// Return:  API Error
+// Purpose: Gets Gage Parameter
+{
+    // Check if Open
+    if(swmm_IsOpenFlag() == FALSE) return(ERR_API_INPUTNOTOPEN);
```

Comment:
I made some recommendations for the new functions in the PR comment.  Specifically for the formatting, the approach we are hoping to take with NEW api functions is to have a single return therein.  Take a look at `swmm_setOutfallStage` as an example.  It is cleaner with a single `return`.

Reply:
Bryant's suggested format would be helpful

Reply:
Thanks @bemcdonnell and @rkertesz. Updated.